src/wp-includes/class-wp-theme-json.php

```diff
@@ -1090,22 +1349,21 @@ private static function get_style_nodes( $theme_json, $selectors = array() ) {
 	 * Merge new incoming data.
 	 *
 	 * @since 5.8.0
+	 * @since 5.9.0 Duotone preset also has origins.
 	 *
 	 * @param WP_Theme_JSON $incoming Data to merge.
 	 */
 	public function merge( $incoming ) {
 		$incoming_data    = $incoming->get_raw_data();
 		$this->theme_json = array_replace_recursive( $this->theme_json, $incoming_data );
 
-		/*
-		 * The array_replace_recursive() algorithm merges at the leaf level.
-		 * For leaf values that are arrays it will use the numeric indexes for replacement.
-		 * In those cases, we want to replace the existing with the incoming value, if it exists.
-		 */
+		// The array_replace_recursive algorithm merges at the leaf level.
+		// For leaf values that are arrays it will use the numeric indexes for replacement.
+		// In those cases, we want to replace the existing with the incoming value, if it exists.
```

Comment:
Core's standard for multiple line comments is with `/* */` convention:

```suggestion
		/*
		 * The array_replace_recursive algorithm merges at the leaf level.
		 * For leaf values that are arrays it will use the numeric indexes for replacement.
		 * In those cases, we want to replace the existing with the incoming value, if it exists.
		 */
```